src/app/component-library/json-dialog/json-dialog/json-dialog.component.html

```diff
@@ -38,7 +38,7 @@
   </div>
   <div class="flex flex-row">
     <button mat-dialog-close mat-button color="default" [disabled]="busy">
-      Cancel
+      Close
```

Comment:
The issue says: "Care that this only should affect the right places, as some places have an additional button, where the cancel makes sense".

Does this also change other dialogs?

If yes could we introduce a mechanism, which defaults to "Cancel" und just set the button text to "Close" if it is passed to the dialog?